src/prompts/ddl/ui.tsx

```diff
@@ -41,17 +59,26 @@ export default class Ui extends Component<UiProps, never> {
         ] = `${offset}px`;
     };
 
-    onBannerConfirm = (config: DdlPromptConfig) => {
-        this.resetBodyStyles();
-        this.props.onBannerConfirm(config);
+    private onBannerConfirm = (config: DdlPromptConfig) => {
+        if (config.uiActions.accept.type === UiActionType.DDL_OPEN_STORE) {
+            this.setState({ requestFpCapture: true });
```

Comment:
Would it be any simpler to have an imperative 'requestFp()' type method on the component and access via a ref? (could use a callback prop or something to give back result)

Reply:
That was ironically the first approach before thinking it was more hacky than going the 'drive by state' way! - completely happy to change this though 👍